### PR TITLE
Stellar now returns to Exchange after a successful sign

### PR DESCRIPTION
**Pull request by fbeutin-ledger**

### CODE REVIEW

**Comment by fbeutin-ledger** [comment]
https://github.com/LedgerHQ/app-exchange/pull/115

**Comment by fbeutin-ledger** [comment]
The warning 
```
src/swap/handle_swap_sign_transaction.c:53:1: error: function declared 'noreturn' should not return [-Werror,-Winvalid-noreturn]
}
^
1 error generated.
```
Is a false positive caused by a missing flag in the SDK API, fix in progress

File: src/swap/handle_swap_sign_transaction.c [hunk 1]
```diff
@@ -34,11 +38,20 @@ bool copy_transaction_parameters(const create_transaction_parameters_t* params)
         return false;
     }
 
+    // Full reset the global variables
     os_explicit_zero_BSS_segment();
+    // Keep the address at wich we'll reply the signing status
```
Comment (xchapron-ledger) [inline src/swap/handle_swap_sign_transaction.c:43]:
```suggestion
    // Keep the address at which we'll reply the signing status
```

**Comment by fbeutin-ledger** [comment]
The issue in `test_stellar_swap_refuse_double_sign` is expected as the test will need an update

**Comment by fbeutin-ledger** [comment]
Zemu tests seem broken because the Speculos is out of date

**Pull request merged**